projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -714,7 +714,7 @@ def default(self, o: Any):
         return json.JSONEncoder.default(self, o)
 
 
-class Worker:
+class Executor:
     """
     Responsible for processing items from the work queue and generating embeddings.
```

Comment:
Line 721 still references "Worker"

Reply:
Should the `WorkerTracking` class be renamed to `ExecutorTracking`?

Reply:
No, worker tracking actually tracks what we call the worker Now. Was misnamed before